test/nuts/seeds/mpd.deploy.seed.ts

```diff
@@ -38,31 +38,31 @@ context('MPD Deploy NUTs [exec: %EXECUTABLE%]', () => {
     describe('--sourcepath', () => {
       it('should deploy all CustomLabels from a single package', async () => {
         await testkit.deploy({ args: `--sourcepath ${path.join('force-app', 'main', 'default', 'labels')}` });
-        await testkit.expect.filesToBeChanged([forceAppLabels]);
+        await testkit.expect.filesToBeDeployed([forceAppLabels]);
```

Comment:
so the original `filesToBeDeployed` is ok for most everything else?

Reply:
yes, it was only an issue when you would deploy `force-app` and then a subset of `force-app` because the subset deploy wouldn't increment the  `RevisionCounter`